oliverli-gubilgut/arrays/sum.go

```diff
@@ -0,0 +1,19 @@
+package main
```

Comment:
Please add a description to the package and the functions to explain what they do. Please follow the form: 

// <PackageName>...